.vscode/launch.json

```
@@ -31,6 +72,8 @@
       ],
       "outputCapture": "std",
       "console": "integratedTerminal",
+      "preLaunchTask": "npm: testenv:run",
+      "postDebugTask": "npm: testenv:stop",
```

I had to add these env vars to get tests passing in the debugger:
```suggestion
      "postDebugTask": "npm: testenv:stop",
      "env": {
        "PGHOST": "localhost",
        "PGUSER": "postgres",
        "PGPASSWORD": "postgres",
      },
```